data-es7.js

```diff
@@ -78,6 +78,16 @@ exports.browsers = {
   firefox41: {
     full: 'Firefox',
     short: 'FF 41',
+    obsolete: true,
```

Comment:
Oh, I sort of misunderstood. Yes, browsers in a separate file is great. But I think there should be some remedy regarding the mess in the test results.